src/Features/Lsif/Generator/CompilerInvocation.cs

```diff
@@ -33,14 +33,14 @@ public CompilerInvocation(Compilation compilation, LanguageServices languageServ
             Options = options;
         }
 
-        public static async Task<CompilerInvocation> CreateFromJsonAsync(string jsonContents)
+        public static async Task<Project> CreateFromJsonAsync(string jsonContents)
         {
             var invocationInfo = JsonConvert.DeserializeObject<CompilerInvocationInfo>(jsonContents);
             Assumes.Present(invocationInfo);
             return await CreateFromInvocationInfoAsync(invocationInfo);
         }
 
-        public static async Task<CompilerInvocation> CreateFromInvocationInfoAsync(CompilerInvocationInfo invocationInfo)
+        public static async Task<Project> CreateFromInvocationInfoAsync(CompilerInvocationInfo invocationInfo)
```

Comment:
Should CompilerInvocation no longer be a constructible type and can the properties and everything be removed?

Reply:
yup, i can likely remove it.

Reply:
done.